.github/workflows/release.yml

```diff
@@ -1,21 +1,32 @@
 name: Release packages and images
 on:
-  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
How does this work @0x2b3bfa0 ?
It runs every X hours but how does it know if it have to refresh the image because of DVC?

Reply:
It runs every 24 hours and updates the images, even when the DVC hasn't changed. It's not especially efficient, but GitHub Actions runs are free and unlimited for public repositories; we promise the latest major version, and that's what users get.

Reply:
We could use the cache action for that. To not have to build the layers. If we move in dockerfile DVC to be the latest.
However! I think that we should taker care of terraform. Did not we had a pin version of terraform?
Also... Have we deprecated docker machine?

Reply:
> _We could use the cache action for that. To not have to build the layers._

Agreed! Nevertheless, this could be a premature optimization. Would improving the build times or resource usage be of any use in this particular case?

> _However! I think that we should taker care of terraform. Did not we had a pin version of terraform?_

No, we [didn't](https://github.com/iterative/cml/blob/8f9d4118f795a89c074721bc712432aae55e467f/docker/Dockerfile#L93-L97), and users shouldn't care at all. Internally, we may want to pin it to a major version, but it doesn't seem to have a specially unstable API.

> _Also... Have we deprecated docker machine?_

Yes, absolutely! Old repositories and tags like `cml:latest` and `cml-py3:latest` keep having it for backwards compatibility, but the new tags rely on Terraform for provisioning and the official Docker >= 19.03 for GPU support, so we're fully aligned with the documentation and don't keep unused packages.

Reply:
> > _We could use the cache action for that. To not have to build the layers._
> 
> Agreed! Nevertheless, this could be a premature optimization. Would improving the build times or resource usage be of any use in this particular case?

It makes a difference for users regularly pulling the image. They will pull fewer layers if we don't keep rebuilding from scratch. Can be a follow-up issue though since I've merged this branch into #530